src/components/Editor/index.js

```diff
@@ -525,10 +525,12 @@ class Editor extends PureComponent {
     const { sourceId } = selectedLocation;
 
     if (bp) {
+      // NOTE: it's possible the breakpoint has slid to a column
+      column = column || bp.location.column;
```

Comment:
Hmm.. I think column should take precedence, so that only when column info's not available we pick it up from bp. When initiated on the gutter, the column is undefined(from what I observed). Perhaps, when we directly click on the bp it could be passed(from codemirror?)?
Please correct me if I'm wrong here :)